.vscode/settings.json

```diff
@@ -1,14 +1,12 @@
 {
```

Comment:
I mainly wanted to remove `"editor.multiCursorModifier": "ctrlCmd"` because I use a different key. Are you OK with removing that line? I don't care much about the other changes.

Reply:
That's fine.

Reply:
Fixed